src/Pages/sangjunkim/Login/Login.js

```diff
@@ -0,0 +1,177 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import "./Login.scss";
+
+class LoginJun extends Component {
+  constructor() {
+    super();
+    this.state = {
+      idValue: "",
+      pwValue: "",
+    };
+  }
+
+  handleIdValue = event => {
+    this.setState({
+      idValue: event.target.value,
+    });
+  };
+
+  handlePwValue = event => {
+    this.setState({
+      pwValue: event.target.value,
+    });
+  };
+
+  goToMain = () => {
+    this.props.history.push("/main-sangjun");
+  };
+
+  render() {
+    return (
+      <div className="LoginJun">
+        <main>
+          <article>
+            <div className="mainLeft">
+              <img
+                src="https://www.instagram.com/static/images/homepage/home-phones.png/43cc71bb1b43.png"
+                alt="showing how instagram works"
+              />
+            </div>
+            <div className="mainRight">
+              <div className="loginInfo">
+                <img
+                  className="logo"
+                  src="/images/sangjunkim/logo_text.png"
+                  alt="instagram logo"
+                />
+                <form className="loginForm">
+                  <div className="loginInput">
+                    <input
+                      className="idForm"
+                      onChange={this.handleIdValue}
+                      type="text"
+                      name="username"
+                      placeholder="전화번호, 사용자 이름 또는 이메일"
+                    />
+                    <input
+                      className="passwordForm"
+                      onChange={this.handlePwValue}
+                      type="password"
+                      name="password"
+                      placeholder="비밀번호"
+                    />
+                    {
+                      <button
+                        className={
+                          this.state.idValue.includes("@") &&
+                          this.state.pwValue.length > 4
+                            ? "loginBtn active"
+                            : "loginBtn inactive"
+                        }
```

Comment:
이 조건이 반복해서 쓰이니까 따로 변수로 빼서 쓸 수 있습니다
그리고 template literals 쓰면 더 줄일 수 있겠네요!
```suggestion
render(){
 return(
 const isValid =   idValue.includes('@') && passValue.length > 5;
 <button
  className={`loginButton ${isValid ? "active" : "inactive"}`}
```

Reply:
수정하였습니다.